distribution/docs/src/main/resources/_contents/_security-framework/security-pdp-authz-realm-contents.adoc

```diff
@@ -0,0 +1,62 @@
+
+==== Security PDP AuthZ Realm
+
+The ${ddf-security} PDP AuthZ Realm exposes a realm service that makes decisions on authorization requests using the attributes stored within the metacard to determine if access should be granted.
+This realm can use XACML and will delegate decisions to an external processing engine if internal processing fails.
+Decisions are first made based on the "match-all" and "match-one" logic.
+Any attributes listed in the "match-all" or "match-one" sections will not be passed to the XACML processing engine and they will be matched internally.
+It is recommended to list as many attributes as possible in these sections to avoid going out to the XACML processing engine for performance reasons.
+If it is desired that all decisions be passed to the XACML processing engine, remove all of the "match-all" and "match-one" configurations.
+The configuration below provides the mapping between user attributes and the attributes being asserted - one map exists for each type of mapping (each map may contain multiple values).
+
+Match-All Mapping:: This mapping is used to guarantee that all values present in the specified metacard attribute exist in the corresponding user attribute.
+Match-One Mapping:: This mapping is used to guarantee that at least one of the values present in the specified metacard attribute exists in the corresponding user attribute.
+
+Match-One Mapping:  This mapping is used to guarantee that at least one of the values present in the specified metacard attribute exists in the corresponding user attribute.
+
+This bundle is not installed by default but can be added by installing the `security-pdp-java` feature.
```

Comment:
security-pdp-java and security-pdp-authz no longer exist and are just called security-pdp-authzrealm and it is installed by default